src/app/legacy/containers/SocialEmbed/index.test.jsx

```diff
@@ -180,6 +180,18 @@ describe('SocialEmbedContainer', () => {
       ).toBeFalsy();
     });
 
+    it('should not render the embed if the ID is invalid', () => {
+      const { container } = render(
+        <SocialEmbedContainer
+          blocks={[youtubeBlockEmbed]}
+          source="https://yout.be/1e05_rwHvOM"
```

Comment:
I can't tell you how long I stared at this not understanding how it should fail given changes to the regex in the helper! haha - I need more coffee, clearly.

Reply:
Haha, maybe I should make it more obvious. I deliberately made it obscure so that I knew the regex definitely worked!

Yea the Instagram changes are actually correct now. The addition of the `if (!id) return null` line highlighted that the tests weren't totally correct.